images/ipfailover/keepalived/lib/failover-functions.sh

```diff
@@ -46,6 +46,19 @@ function setup_failover() {
     echo "ERROR: Module ip_vs is NOT available."
   fi
 
+  # When the DC supplies an (non null) iptables chain
+  # (OPENSHIFT_HA_IPTABLES_CHAIN) make sure the rule to pass keepalived
+  # multicast (224.0.0.18) is in the table.
+  chain="${OPENSHIFT_HA_IPTABLES_CHAIN:-""}"
+  if [[ -n ${chain} ]]; then
+    echo "  - check for iptables rule for keepalived multicast (224.0.0.18) ..."
+    if ! iptables -S | grep 224.0.0.18 > /dev/null ; then
```

Comment:
If something were to go wrong with this, you'll get stuff from `stderr` popping out -- maybe this is intended? If not, `> /dev/null 2>&`.